README.md

```diff
@@ -95,6 +98,33 @@ ios-app Info.plist:
 ```
 in array should be added all used languages.
 
+For JS/Browser you will have to add the following file to your JS-Projects [webpack.config.d](https://kotlinlang.org/docs/js-project-setup.html#webpack-configuration-file) folder:
+```js
+const path = require('path');
+
+/*
+ * your current path is root-project-folder/build/js/packages/project_name-js
+ * this path has to link to root-project-folder/your-common-project/build/generated/moko/jsMain/resources
+ * 
+ * So for example "../../../../common/build/generated/moko/jsMain/resources"
+ */
+const mokoResourcePath = path.resolve("REPLACE_WITH_YOUR_PATH_AS_IN_COMMENT")
+
+config.module.rules.push(
+    {
+        test: /\.(.*)/,
+        include: [
+            path.resolve(mokoResourcePath)
+        ],
+        type: 'asset/resource'
+    }
+);
+
+config.resolve.modules.push(
+    path.resolve(mokoResourcePath)
+)
```

Comment:
i think this part can be done automatically by gradle plugin

Reply:
I don't think so. I've looked at this for quite some time. At least there is nothing in the documentation. And everything else feels like a hack. I've had a discussion about this in the kotlin slack: https://kotlinlang.slack.com/archives/C0B8L3U69/p1641383346450900

Reply:
we already implement generation by plugin